packages/runtime/src/helpers/edge.ts

```diff
@@ -98,6 +124,22 @@ const getMiddlewareBundle = async ({
 }): Promise<string> => {
   const { publish } = netlifyConfig.build
   const chunks: Array<string> = [preamble]
+
+  if ('wasm' in edgeFunctionDefinition) {
+    for (const { name, filePath } of edgeFunctionDefinition.wasm) {
+      const wasm = await fs.readFile(join(publish, filePath))
```

Comment:
Can we use the file system in Edge functions now? I thought for some reason we didn't have access to `Deno.fs*`?

Reply:
This is run during the build, so it uses Node

Reply:
Doh! I could've sworn I saw a Deno.readFile. Late in the day PR reviews. 🫠